config/default/filter.format.minimal_html.yml

```diff
@@ -17,7 +17,7 @@ filters:
     status: true
     weight: -10
     settings:
-      allowed_html: '<em> <strong> <cite> <blockquote cite> <code> <ul type> <ol start type> <li> <dl> <dt> <dd> <h2 id> <h3 id> <h4 id> <h5 id> <h6 id> <hr> <p class> <span class> <table class> <a href hreflang data-entity-substitution data-entity-type data-entity-uuid title class>'
+      allowed_html: '<em> <strong> <cite> <blockquote cite> <code> <ul type> <ol start type> <li> <dl> <dt> <dd> <h2 id> <h3 id> <h4 id> <h5 id> <h6 id> <hr> <p class=""> <span class=""> <table class=""> <a href hreflang data-entity-substitution data-entity-type data-entity-uuid title class=" hs-button hs-button--big hs-secondary-button hs-more-link hs-external-link hs-file-link hs-pdf-link hs-private-link">'
```

Comment:
So with the minimal html formatting, the WYSIWYG will actually strip out any html tags that aren't allowed to be added.  Essentially protecting the WYSIWYG from html entry that you do not want.  Adding it to the the "allowed html" allows those classes or tags to be allowed and therefore not stripped out by the WYSIWYG upon saving.  

This is especially important if classes or tags were added using something like "Full HTML" but then a different user that doesn't have "FULL HTML" access and is set to Minimal or something similar. It will keep the WYSIWYG editor from stripping those previously added html classes or tags out of it upon that user saving the node.